### PR TITLE
add angular-tree-control v0.2.26 with npm autoupdate

### LINKED ISSUE #7470: [Request] Add angular-tree-control
**Library name:** angular-tree-control
**Git repository url:** https://github.com/wix/angular-tree-control
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:** http://wix.github.io/angular-tree-control/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7470 
@kennynaoh Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/wix/angular-tree-control
 * Official website (optional, not the repository): https://wix.github.io/angular-tree-control
 * NPM package url (optional): https://www.npmjs.com/package/angular-tree-control
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 447
   - Count of watchers: 323
   - Count of forks: 200
 * NPM download stats (optional):
   - Downloads in the last day: 91
   - Downloads in the last week: 546
   - Downloads in the last month: 2387

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7470, cc @yoavaa

### CODE REVIEW

File: ajax/libs/angular-tree-control/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "type": "git",
+    "url": "git://github.com/wix/angular-tree-control.git"
+  },
+  "keywords": [
+    "angular-tree-control",
+    "Angular JS Tree",
+    "tree control component"
+  ],
+  "author": "Yoav Abrahami",
+  "license": "ISC",
```
Comment (kennynaoh) [inline ajax/libs/angular-tree-control/package.json]: @pvnr0082t 
I see the "MIT" license in [LICENSE](https://github.com/wix/angular-tree-control/blob/master/LICENSE)
Maybe you should confirm it.
Thanks.

**Comment by pvnr0082t** [comment]
@kennynaoh I have checked license. It's ```MIT```. And the latest version is v0.2.26. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I am not sure whether the file ```angular-tree-control.min.js.map``` should be added or not?
Please help me confirm this.
Thanks.

**Comment by pvnr0082t** [comment]
I use ```--no-map``` when doing minify.

**Comment by PeterDaveHello** [comment]
@pvnr0082t any reason to disable the map?

**Comment by pvnr0082t** [comment]
@PeterDaveHello I don't know the difference with and without map. Is that better to enable map?

**Comment by PeterDaveHello** [comment]
@pvnr0082t It's better to have a map file for debug and development works, see https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map

**Comment by pvnr0082t** [comment]
@kennynaoh I have re-minified files with map. Please help me review it again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

File: ajax/libs/angular-tree-control/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "author": "Yoav Abrahami",
+  "license": "MIT",
+  "npmName": "angular-tree-control",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "angular-tree-control*",
+        "+(css|images)/!(sample).+(css|png|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-tree-control/package.json]: I don't think combine css and images in the same rule is a good idea, they have difference naming pattern.

**Comment by pvnr0082t** [comment]
@kennynaoh I have update ```filemap``` in package.json. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Pull request merged**